modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/portlet/AdminPortlet.java

```diff
@@ -159,6 +147,12 @@ protected void activate(Map<String, Object> properties) {
 				ReportsEngineAdminWebConfiguration.class, properties);
 	}
 
+	@Reference
+	protected DefinitionLocalService definitionLocalService;
```

Comment:
@W2jx Please update the commit message to let people know this class has a subclass so we need to keep these protected.

Reply:
I will fix it.